backend/kernelCI_app/views/treeCommitsHistory.py

```diff
@@ -218,8 +220,13 @@ def _process_rows(self, rows: Dict) -> None:
         sanitized_rows = self.sanitize_rows(rows)
 
         for row in sanitized_rows:
+            hardware_filter = row["hardware_compatibles"]
```

Comment:
isn't this variable referring to the current hardware of a row? I don't think hardware\__filter_ is a good name for it